### ISSUE TITLE
[BUG] Installation with pytest producing warning from pip.

### ISSUE THREAD

**Issue by vlade11115** [schemathesis/schemathesis#858]
**Describe the bug**
Hi. Currently schemathesis pins `attrs = "^19.1"` as [dependency](https://github.com/schemathesis/schemathesis/blob/cb995e5147019c6519bde34bc2816bcd7ec7a0b3/pyproject.toml#L33).
And pytest pins `attrs>=19.2.0` as [dependency](https://github.com/pytest-dev/pytest/blob/843bca8c0cf89405d426b40133772d43551f668d/setup.cfg#L42).
Because of this, right now `pip install pytest schemathesis --quiet` rase error (`--quiet` added for less useless logs).

```
pip install pytest schemathesis --quiet
ERROR: schemathesis 2.7.5 has requirement attrs<20.0,>=19.1, but you'll have attrs 20.3.0 which is incompatible.
WARNING: You are using pip version 20.1.1; however, version 20.2.4 is available.
You should consider upgrading via the '/Users/vlad/venv/bin/python3 -m pip install --upgrade pip' command.
```

**To Reproduce**
1. Run this command `pip install pytest schemathesis --quiet` in fresh venv.
2. See error `ERROR: schemathesis 2.7.5 has requirement attrs<20.0,>=19.1, but you'll have attrs 20.3.0 which is incompatible.`

**Expected behavior**
I expect no warnings from pip when schemathesis and pytest are installed in the same venv.

**Environment (please complete the following information):**
 - OS: Mac OS, AWS Linux
 - Python version: 3.8.5
 - Schemathesis version: 2.7.5

**Additional context**
It just a warning from pip, but annoing one.

**Comment by Stranger6667** [comment]
Hi! Thank you for the detailed report, will fix in a few minutes!

**Comment by Stranger6667** [comment]
Fixed in `2.7.6`
